pkg/utils/statusCheck.go

```diff
@@ -36,3 +39,20 @@ func RunnerPodStatus(runnerNamespace string, engineName string, client *kubernet
 
 	return 0, nil
 }
+
+//DeploymentRunningStatus checks running status of deployment for a certain delay and retries
+func DeploymentRunningStatus(namespace, deploymentName string) error {
+	sampleApp, _ := chaosTypes.Client.AppsV1().Deployments("default").Get("testapp1", metav1.GetOptions{})
```

Comment:
Can you use namespace & deploymentName args instead of hardcoded them?

Reply:
ok done